src/Avalonia.Controls/Primitives/RangeBase.cs

```diff
@@ -75,10 +75,18 @@ public double Minimum
 
             set
             {
-                value = ValidateMinimum(value);
-                SetAndRaise(MinimumProperty, ref _minimum, value);
-                Maximum = ValidateMaximum(Maximum);
-                Value = ValidateValue(Value);
+                ValidateDouble(value, "Minimum");
+
+                if (IsInitialized)
+                {
+                    SetAndRaise(MinimumProperty, ref _minimum, value);
+                    Maximum = ValidateMaximum(Maximum);
```

Comment:
Shouldn't this be `Minimum` ?

Reply:
No, I don't think so - we've already set `Minimum` with `SetAndRaise`. This line is coercing `Maximum` to be greater than `Minimum`.